test/integration/test_connection.yml

```diff
@@ -0,0 +1,35 @@
+- hosts: all
+  gather_facts: no
+  serial: 1
+  tasks:
+
```

Comment:
The inventory group "skip-during-build" lists ssh and paramiko_ssh, so those won't run automatically when running ```make test_connection```, but you can override that with TEST_FLAGS.

If we can get a local ssh server running in the docker containers during build in the future we could allow those to run in that scenario.